include/devicetree/zephyr.h

```diff
@@ -64,9 +64,9 @@
 	DT_LABEL(DT_CHOSEN(zephyr_flash_controller))
 #endif
 
-#if DT_NODE_HAS_PROP(DT_CHOSEN(zephyr_can_primary), label)
-#define DT_CHOSEN_ZEPHYR_CAN_PRIMARY_LABEL \
-	DT_LABEL(DT_CHOSEN(zephyr_can_primary))
+#if DT_NODE_HAS_PROP(DT_CHOSEN(zephyr_canbus), label)
+#define DT_CHOSEN_ZEPHYR_CANBUS_LABEL \
+	DT_LABEL(DT_CHOSEN(zephyr_canbus))
```

Comment:
I'm wondering why we need these macro definitions at all. They look like a leftover from something around Zephyr 2.1 where the `DT_CHOSEN(node)` macros didn't exist yet. Can't we just replace with the direct `DT_*` calls?

Reply:
I agree, but I would like to leave that change to a separate PR.